### PR TITLE
DOC Docs on how to use learning rate schedulers

### PR DESCRIPTION
**Pull request by dnouri**
More hints for confused people like me. ;-)

### CODE REVIEW

File: docs/user/callbacks.rst [hunk 1]
```diff
@@ -230,3 +230,34 @@ starting with ``f_``:
 
 Please refer to :ref:`saving and loading` for more information about
 restoring your network from a checkpoint.
+
+
+Learning rate schedulers
+------------------------
+
+The :class:`.LRScheduler` callback allows the use of the various
+learning rate scheduler defined in :mod:`torch.optim.lr_scheduler`,
```
Comment (BenjaminBossan) [inline docs/user/callbacks.rst]:
```suggestion
learning rate schedulers defined in :mod:`torch.optim.lr_scheduler`,
```

File: docs/user/callbacks.rst [hunk 2]
```diff
@@ -230,3 +230,34 @@ starting with ``f_``:
...
+such as :class:`~torch.optim.lr_scheduler.ReduceLROnPlateau`, which
+allows dynamic learning rate reducing based on a given value to
+monitor, or :class:`~torch.optim.lr_scheduler.CyclicLR`, which cycles
+the learning rate between two boundaries with a constant frequency.
+
+As with other callbacks, you can use `set_params`, and thus search
```
Comment (BenjaminBossan) [inline docs/user/callbacks.rst]: You talk about grid search here but the code example just shows the normal use. Therefore, I would start this paragraph with the last sentence, "Here's a network ...", followed by the code example, then put the part "As with other callbacks..." below the code example.

Comment (dnouri) [reply docs/user/callbacks.rst]: I followed your suggestion and in addition added a grid search code example below.

**Comment by BenjaminBossan** [comment]
Thanks, especially for adding the grid search example :)
